Content/NPCs/Vitric/Gauntlet/FlyingGruntConstruct.cs

```diff
@@ -509,5 +522,22 @@ public override void SetBestiary(BestiaryDatabase database, BestiaryEntry bestia
 				new FlavorTextBestiaryInfoElement("One of the Glassweaver's constructs. An already formidable duelist, made airborne - speed is war.")
 			});
 		}
+
+		public override void SafeSendExtraAI(BinaryWriter writer)
+		{
+			//frames are being used like cursed timers, maybe TODO: rework this
```

Comment:
Would it be a difficult refactor to simply pull these out to their own timer variables? We already have to sync them here manually.

Reply:
The annoying part is they are directly related to spritesheet frames and ai code is pulling from those like states

Reply:
I don't think i'm gonna crack this one open on this PR i'll leave it to someone else it looks like a lot of lines of code to check and recheck

Reply:
Fair, lets leave this as an enhancement to target later